OpenRA.Game/Network/OrderManager.cs

```diff
@@ -149,8 +154,7 @@ public void TickImmediate()
 						immediatePackets.Add((clientId, packet));
 					else
 					{
-						activeClients.Add(clientId);
-						pendingPackets.GetOrAdd(clientId).Enqueue(packet);
+						pendingPackets[clientId].Enqueue(packet);
```

Comment:
This will crash if any client packets are already waiting in the connection buffer to be processed in this same tick as the StartGame. Moving the `immediatePackets` processing inline in the case above fixes this (and hopefully shouldn't break anything else...)

Repro case:
 * Start a local server with two clients
 * Pause the second (non-host) client using ctrl-z or IDE debugger
 * Use the "Start Game" button on the host client to start the game
 * Wait for the host to have loaded and entered the world (will be frozen on tick 0 due to the other client stalling)
 * Unfreeze the second client, which will now receive the StartGame and Ticks 0-Latency orders from the first client in the same tick.

Reply:
We should also use `TryGetValue` here and write to debug.log and drop the packet if the client has already disconnected.